src/main/java/mods/railcraft/client/render/carts/CartContentRendererMaintenance.java

```diff
@@ -25,6 +26,7 @@
     private static final ModelBase LAMP = new ModelMaintenanceLamp();
     private static final ResourceLocation LAMP_ON_TEX = new ResourceLocation(RailcraftConstants.CART_TEXTURE_FOLDER + "cart_maint_lamp_on.png");
     private static final ResourceLocation LAMP_OFF_TEX = new ResourceLocation(RailcraftConstants.CART_TEXTURE_FOLDER + "cart_maint_lamp_off.png");
+    private static final ResourceLocation LAMP_DISABLED_TEX = new ResourceLocation(RailcraftConstants.CART_TEXTURE_FOLDER + "cart_maint_lamp_disabled.png");
```

Comment:
You didn't add this texture yet.

Reply:
It somehow didn't make it into the commit. I'll fix that now